adapters/repos/db/vector/hnsw/startup.go

```diff
@@ -197,6 +203,44 @@ func (h *hnsw) restoreFromDisk() error {
 	return nil
 }
 
+func (h *hnsw) restoreDocMappings() error {
+	prevDocID := uint64(0)
+	relativeID := uint64(0)
+	maxNodeID := uint64(0)
+	for _, node := range h.nodes {
+		if node == nil {
+			continue
+		}
+		nodeIDBytes := make([]byte, 8)
+		binary.BigEndian.PutUint64(nodeIDBytes, node.id)
+		docIDBytes, err := h.store.Bucket(h.id + "_mv_mappings").Get(nodeIDBytes)
+		if err != nil {
+			return errors.Wrap(err, "get docIDBytes")
```

Comment:
We should return a better error message. Something like `failed to get %s_mv_mappings from the bucket`